10314.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- MIT License
+Copyright (c) 2019 OpenLX SP Ltd
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, 
+including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A 
+PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, 
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+	<Object ObjectType="MODefinition">
+		<Name>Particulates</Name>
+		<Description1><![CDATA[This object should be used with a particulate air quality sensor to report a particulates measurement.  It also provides resources for minimum/maximum measured values and the minimum/maximum range that can be measured by the particulate sensor. An example measurement unit is microgrammes per cubic metre ug/m3.]]></Description1>
```

Comment:
Currently we don't have ug/m3 [unit](https://www.iana.org/assignments/senml/senml.xhtml#senml-units). What's the SI unit here? kg/m3? Are other SI units used in this context? If no, we could fix the unit in the schema.

Reply:
Particulates are only specified in ug/m3 and (this is what is used when the data is displayed, and generated by the sensors). So SI unit is kg/m3 but no one ever talks about anything other than ug/m3.
This is perhaps part of a bigger discussion -- hypothetically if I'm a programmer writing an application that consumes/displays data from an resource, I'd prefer it if the object could give me a sensible unit to display rather than having to have domain knowledge to convert it. 

Should I put kg/m3 then?

Reply:
This is a very valid point and something we are planning to discuss next week in the OMA SpecWorks f2f meeting. If you don't mind waiting until after next week, we should be able to give better answer then.

Reply:
That's fine, I appreciate it's a tricky one.
I think it can't be deduced just from the scaling factor of the data. I've not thought too deeply, but e.g.
 ug/m3 and kg/km3 are both kg/m3 scaled by 1e+9.
Thanks for your help so far.